shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -54,76 +59,120 @@ public MetricsCommand(MetaMasterClient metaMasterClient, PrintStream printStream
    * @return 0 on success, 1 otherwise
    */
   public int run() throws IOException {
-    Map<String, MetricValue> metricsMap = new TreeMap<>(mMetaMasterClient.getMetrics());
-    Set<String> operations = new HashSet<>();
-    operations.add(MasterMetrics.DIRECTORIES_CREATED);
-    operations.add(MasterMetrics.FILE_BLOCK_INFOS_GOT);
-    operations.add(MasterMetrics.FILE_INFOS_GOT);
-    operations.add(MasterMetrics.FILES_COMPLETED);
-    operations.add(MasterMetrics.FILES_CREATED);
-    operations.add(MasterMetrics.FILES_FREED);
-    operations.add(MasterMetrics.FILES_PERSISTED);
-    operations.add(MasterMetrics.NEW_BLOCKS_GOT);
-    operations.add(MasterMetrics.PATHS_DELETED);
-    operations.add(MasterMetrics.PATHS_MOUNTED);
-    operations.add(MasterMetrics.PATHS_RENAMED);
-    operations.add(MasterMetrics.PATHS_UNMOUNTED);
-
-    mPrintStream.println("Alluxio logical operations: ");
-    for (Map.Entry<String, MetricValue> entry : metricsMap.entrySet()) {
-      String key = entry.getKey();
-      if (operations.contains(key)) {
-        printIndentedMetrics(key, entry.getValue());
-      }
-    }
+    Long bytesReadLocal = mMetricsMap.getOrDefault(MetricsSystem.getClusterMetricName(
```

Comment:
declare as close to usage as possible

Reply:
I declare those properties here is because the bytesReadLocal, bytesReadAlluxio and BytesReadUfsAll metrics info will be print later and printMetrics() will delete those metrics from the mMetricsMap after printing the info.